web/src/views/DandisetLandingView/DandisetDetails.vue

```diff
@@ -173,12 +173,12 @@
         <v-col cols="12">
           <v-chip
             v-for="owner in limitedOwners"
-            :key="owner.id"
+            :key="owner.id /* TODO remove this */ || owner.username"
             color="light-blue lighten-4"
             text-color="light-blue darken-3"
             class="font-weight-medium ma-1"
           >
-            {{ owner.login }}
+            {{ owner.login /* TODO remove this */ || owner.username }}
```

Comment:
Is this currently to deal with both girder and the new API at the same time?

Reply:
Yep. The Girder user object has a `login` property while the Django user object has a `username`. It was easier to just render like this than to do a proper girderization of the object.